VISION.md

```diff
@@ -0,0 +1,86 @@
+# API Improvement Proposals: Mission and Vision
+
+## Mission
+
+Help organizations build consistent, uniform remote application programming
+interfaces and clients.
+
+## Vision
+
+API Improvement Proposals help producers of APIs delight their users with API
+styleguide tooling, API best practices and validation tooling, and client
+generators.
```

Comment:
I like the message, but the phrasing feels a bit awkward to me.  I think there are two pieces, at a very high level:

1.  Better API design via mature guidance, benefiting both producers and consumers.
2.  Tools to support API production (linter) and API consumption (generated client libraries, maybe generated docs one day).